core/domain/topic_services.py

```diff
@@ -1342,6 +1342,29 @@ def check_can_edit_topic(
     return False
 
 
+def check_can_edit_question(
```

Comment:
Can this function be named in a more declarative way? Something like `is_question_editable_by(user, topic_rights)` or `can_edit_question(user, topic_rights)`?